packages/admin/cms-admin/src/dam/Table/fileUpload/useFileUpload.tsx

```diff
@@ -111,6 +111,10 @@ export const useFileUpload = (options: UploadFileOptions): FileUploadApi => {
             if (extensions) {
                 acceptObj[mimetype] = extensions.map((extension) => `.${extension}`);
             }
+
+            if (mimetype === "application/x-zip-compressed") {
```

Comment:
Do we need this special treatment in the admin? The file has been uploaded, but failed in the API?

Reply:
Dropzone writes all mimetypes and extensions into the input file field `accept` parameter (https://www.w3schools.com/tags/att_input_accept.asp). This parameter supports both, mimetypes and extensions. 

So the upload works because .zip is in the list (although `application/x-zip-compressed` isn't).

I think it's safer to keep this special treatment (who knows, maybe there are differences between browsers regarding how they handle the accept parameter?)